.github/workflows/ci.yml

```diff
@@ -81,7 +81,9 @@ jobs:
           if [ "$RUNNER_OS" == "macOS" ]; then  # Avoid numba/OpenMP segfault in CVMDrift (https://github.com/SeldonIO/alibi-detect/issues/648)
             export NUMBA_THREADING_LAYER="workqueue"
           fi
-          pytest alibi_detect
+          pytest --randomly-seed=0 alibi_detect
+          # Note: The pytest-randomly seed is fixed at 0 for now. Once the legacy np.random.seed(0)'s 
+          # are removed from tests, this can be removed, allowing all tests to use random seeds.
```

Comment:
Not sure I understand this change, also isn't this independent of the notebook tests?